package.json

```diff
@@ -29,6 +29,7 @@
     "lint:prettier": "prettier --check \"{**/*,*}.{js,ts,jsx,tsx}\"",
     "lint:stylelint": "stylelint '**/*.{css,scss}'",
     "lint:ts": "tsc --pretty --noEmit -p ./scripts && yarn workspaces run lint:ts",
+    "load-content-types": "npx ts-node ./scripts/content-types/load-content-types",
```

Comment:
Oops! 😅

Reply:
We store this script outside of the web package so I moved it here.